### ISSUE TITLE
support for ionic angular 2

### ISSUE THREAD

**Issue by frndxyz** [kunder-lab/cl.kunder.webview#26]
any demo to work with ionic 2 , angular 2?

thanks..

**Comment by MichaelTague** [comment]
Here is what I did to make it work in Ionic 2.   By way of example, here it is added to a simple Ionic 2 app:

    ionic start ionictest blank           // Builds a simple Hello World style app.
    cd ionictest
    ionic cordova platform add android    // Only works on Android or iOS
    ionic cordova run android

At this point I got an error about Gradle not being in the path.  This seems to be a recent bug.   This line will fix it (at some point in the future this won't be necessary); then run again:

    cordova platform update android@6.1.1
    ionic cordova run android

So far, we haven't touched the plugin.  Assuming all is good, let's add the plugin:

    ionic cordova plugin add https://github.com/kunder-lab/cl.kunder.webview.git

Now edit pages/home/home.html to add a button (after the "If you get lost ..." paragraph):

    
    <button ion-button (click)="launch('http://MichaelTague.com')">
        Simple Web Site
    </button>

And then in pages/home/home.ts, add this right after the imports:

    declare var webview: any;

and then inside the "export class HomePage" block, put this:

    launch(url: string) {
        webview.Show(url);
    }

The "declare" tells TypeScript that is is OK to refer to "webview" without it otherwise being imported or instantiated.   Note: there is no import of the plugin.   The webview.Show(...) will open this URL in a second webview.  It seems to have no trouble loading the HTML and related files such as an image.  However, if you touch a link, the webview will open a browser.

As for how to get it to talk to an existing cordova plugin, I'm still working on that!  Maybe someone else will comment.

Good luck, Michael Tague (tague@win.net).
